hs_app_timeseries/models.py

```diff
@@ -20,29 +20,17 @@ class Site(AbstractMetaDataElement):
     elevation_datum = models.CharField(max_length=50, null=True, blank=True)
     site_type = models.CharField(max_length=100, null=True, blank=True)
 
+    def __str__(self):
+        return self.site_name
+
     def __unicode__(self):
-        self.site_name
+        return unicode(str(self))
```

Comment:
So the ```__str__()``` function be like this?

    def __str__(self):
         return force_bytes(self.site_name, errors="ignore")

I am also confused reading this (https://docs.djangoproject.com/en/1.8/ref/unicode/#models) where it says no need to have a ``` __str__()```